src/cheatsheet.jl

```diff
@@ -0,0 +1,413 @@
+using LinearAlgebra, GeometryBasics
+using Makie.SparseArrays
+
+function cheatsheet_3d(rn, testimage)
```

Comment:
```suggestion
function cheatsheet_3d(rn = randn(10), testimage = Makie.logo)
```
so that you can just call `Makie.cheatsheet_3d()`

Reply:
yeah, i have to run and quickly wanted to spin up CI for the benchmarks, so this isn't the final version ;)